gtyped/codecs.go

```diff
@@ -0,0 +1,46 @@
+package gtyped
+
+import (
+	"fmt"
+
+	"github.com/lovoo/goka"
+)
+
+type GCodec[V any] interface {
+	GEncode(value V) (data []byte, err error)
+	GDecode(data []byte) (value V, err error)
+}
+
+type codecBridge[T any] struct {
```

Comment:
The GOF name is adapter. 
So maybe?
```suggestion
type codecAdapter[T any] struct {
```

Also with the ctor func below.

Reply:
top, thanks